packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -138,32 +140,86 @@ export class PostgresClient<
         this.config = config;
     }
 
+    private getSQLWithMetadata(sql: string, tags?: Record<string, string>) {
+        let alteredQuery = sql;
+        if (tags) {
+            alteredQuery = `${alteredQuery}\n-- ${JSON.stringify(tags)}`;
```

Comment:
I'm not sure what this is doing

Reply:
This logic was introduced in this [PR](https://github.com/lightdash/lightdash/pull/5784), I just moved it to a private function.